platform/email/platform-email-impl/pom.xml

```diff
@@ -72,6 +72,17 @@
             <groupId>ddf.platform.util</groupId>
             <artifactId>platform-util</artifactId>
         </dependency>
+        <dependency>
+            <groupId>ddf.security.encryption</groupId>
+            <artifactId>security-encryption-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>${mockito.version}</version>
+            <scope>test</scope>
```

Comment:
✏️ No need to redeclare mockito. See https://github.com/codice/ddf/blob/2.26.x/pom.xml#L1222-L1225

Reply:
Good call, I'll update this!